packages/runtime/src/module/index.ts

```diff
@@ -32,7 +32,7 @@ class Module {
     const remoteEntryExports = await getRemoteEntry({
       remoteInfo: this.remoteInfo,
       remoteEntryExports: this.remoteEntryExports,
-      createScriptHook: (url: string, attrs: any) => {
+      createScriptHook: (url, attrs) => {
```

Comment:
was this because we have a implicit dependency ts lint rule? if not why were the typings dropped?

Reply:
typing comes from `getRemoteEntry` type which was added now, so these typings were redundant :)